### PR TITLE
Core/App: Change - Unable to partial extend app. ED-1027.

### PR DESCRIPTION
**Pull request by iNewLegend**

### CODE REVIEW

**Review by matipojo — APPROVED** [review]

File: core/app/assets/js/ui/modal/modal.js [hunk 1]
```diff
@@ -70,14 +71,14 @@ class Modal extends React.Component {
...
+		const { modalProps, children, icon } = this.props;
 		return modalProps.show ? (
 			<div className="eps-modal__overlay" onClick={ this.closeModal }>
 				<div className="eps-modal" ref={ this.modalRef } >
 					<Grid container className="eps-modal__header" justify="space-between" alignItems="center">
 						<Grid item>
-							<Icon className="eps-modal__icon eicon-info-circle"/>
-							{ this.props.title }
+							<Icon className={`eps-modal__icon ${ icon }`} />
+							<span className="title">{ this.props.title }</span>
```
Comment (matipojo) [inline core/app/assets/js/ui/modal/modal.js:81]: Use `Typography`

**Review by matipojo — CHANGES_REQUESTED** [review]

File: core/app/assets/js/ui/modal/modal.js [hunk 2]
```diff
@@ -30,7 +30,7 @@ export default class ModalProvider extends React.Component {
 		return (
 			<>
 				<Button { ... this.props.toggleButtonProps } onClick={ this.state.showModal } />
-				<Modal modalProps={this.state} title={ this.props.title }>
+				<Modal modalProps={this.state} title={ this.props.title } icon={ this.props.icon }>
```
Comment (joshmarom) [inline core/app/assets/js/ui/modal/modal.js:33]:
```suggestion
				<Modal modalProps={this.state} {...this.props}>
```

**Review by joshmarom — COMMENTED** [review]
Can you describe in a few words what was the objective and what was done to accomplish that...

I see some changes in SCSS bundling and RTL can you explain abut that too, and how it all comes together?

**Comment by iNewLegend** [comment]
> Can you describe in a few words what was the objective and what was done to accomplish that...
> 
> I see some changes in SCSS bundling and RTL can you explain abut that too, and how it all comes together?

The objective was: to create custom modal.
What was done to accomplish that:
1. minimal app-base for the requirements.
2. some of the components was not fully extend able, like modal - **eg:** we wanted to change the icon, title, etc.

"I see some changes",  for the minimal app-base css.

File: core/app/assets/js/ui/modal/modal.js (hunk 1 above)
Comment (iNewLegend) [reply core/app/assets/js/ui/modal/modal.js]: Handled.

File: core/app/assets/js/ui/modal/modal.js (hunk 2 above)
Comment (iNewLegend) [reply core/app/assets/js/ui/modal/modal.js]: Handled.

**Review by matipojo — APPROVED** [review]

**Comment by joshmarom** [comment]
I can't find anything wrong with this PR

**Pull request merged**